app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -218,81 +241,32 @@ public function __construct(
      */
     protected function _construct()
     {
-        $this->_init(\Magento\Sitemap\Model\ResourceModel\Sitemap::class);
+        $this->_init(SitemapResource::class);
     }
 
     /**
      * Get file handler
      *
      * @return \Magento\Framework\Filesystem\File\WriteInterface
-     * @throws \Magento\Framework\Exception\LocalizedException
+     * @throws LocalizedException
      */
     protected function _getStream()
     {
         if ($this->_stream) {
             return $this->_stream;
         } else {
-            throw new \Magento\Framework\Exception\LocalizedException(__('File handler unreachable'));
+            throw new LocalizedException(__('File handler unreachable'));
         }
     }
 
     /**
-     * Add a sitemap item to the array of sitemap items
```

Comment:
Please restore these two public methods:
- they are in the class marked as @api
- the class will be released as is in 2.2 release
- current pull request will get to the 2.3 release